firefox-ios/Client/ApplicationHelper.swift

```diff
@@ -46,4 +47,18 @@ struct DefaultApplicationHelper: ApplicationHelper {
             open(url)
         }
     }
+
+    func closeTab(_ url: URL) async {
+        // We need to get all the windows/tabs and then find the url that was requested to close
+        await UIApplication.shared.connectedScenes.forEach { scene in
```

Comment:
This line shouldn't be necessary, AFAICS? We should be able to simply iterate `allWindowTabManagers().forEach` I believe.